gammapy/background/off_data_background_maker.py

```diff
@@ -203,3 +224,81 @@ def save_models(self, modeltype):
         """
         for ngroup in range(self.ntot_group):
             self.save_model(modeltype, ngroup)
+
+    def make_bkg_model_link(self, data_store, modeltype, out_dir_background_model=None, filename_obs_group_table=None):
+        """
+
+        Parameters
+        ----------
+        data_store : `~gammapy.data.DataStore`
+            Create a `~gammapy.data.DataStore` for the runs for which ones we want to compute a background model
```

Comment:
> Create a `~gammapy.data.DataStore` for the runs for which ones we want to compute a background model

change to just

> Data store

specifically remove the statement "Create" because that sounds like the method would instantiate a data store, but it is just passed an existing one and doesn't modify it (right?).